firefox/MozillaFirefox.spec

```diff
@@ -44,7 +43,7 @@
 %define do_profiling   0
 
 # upstream default is clang (to use gcc for large parts set to 0)
-%define clang_build    1
+%define clang_build 0
```

Comment:
Need to keep clang for Tumbleweed

Reply:
Probably because you have a newer gcc there? Seems to build fine with gcc on the SLE/Leap side.